compiler/src/iree/compiler/Dialect/TensorExt/IR/TensorExtOps.td

```diff
@@ -307,6 +307,39 @@ def IREETensorExt_DispatchWorkgroupCountFromSliceOp :
   }];
 }
 
+def IREETensorExt_DispatchWorkgroupCountSplitReductionModifierOp :
+    IREETensorExt_PureOp<"dispatch.workgroup_count_split_reduction_modifier"> {
+  let summary = [{
+    Modifies the workgroup count calculation to account for split reductions.
+  }];
+  let description = [{
+    A split reduction dispatch contains two parts: a `scf.forall` and linalg
+    compute ops within the scf op. `workgroup_x`, `workgroup_y`, and
```

Comment:
nit: I'd replace linalg compute ops with TilingInterface ops or compute ops, because split reduction is also available for LinalgExt ops. If we look at the basic, it should be available for any reduction op that implements tiling interface.